### DISCUSSION TITLE
Add styles to button states without overwriting the existing styles?

### DISCUSSION THREAD

**Discussion by keithburgie** [chakra-ui/chakra-ui#7213] [Q&A]
If I add the following to a `Button` component...
```
_active: {
  transform: "scale(.95)",
},
_hover: {
  transform: "scale(1.05)",
}
```
... I lose the default background color behavior. Is there a way to just add these additional styles without losing the rest?

**Comment by ziyuez** [comment]
It looks like this might be caused by typos in your code. I removed some erroneous curly brackets -- try the following and see if this solves your problem!
```
_active: {
  transform: "scale(.95)",
},
_hover: {
  transform: "scale(1.05)",
}
```

**Reply by keithburgie** [reply]
LMAO. I hope it wasn't actually like my original post at the time. I'm assuming the file would have thrown an error. Thank you for your response, though.

**Comment by AnonymousXC** [comment]
You could extend the theme and make a new variant of the `Button` component.

[May be it can help](https://chakra-ui.com/docs/styled-system/component-style#styling-single-part-components)
